gossip/src/restart_crds_values.rs

```diff
@@ -122,6 +141,64 @@ impl RestartLastVotedForkSlots {
     }
 }
 
+impl Sanitize for RestartHeaviestFork {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
```

Comment:
can we sanitize the wallclock like the other crds values?
also probably that should be done for the previous restart crds value as well.

Reply:
Done.